Update annotations.xml

pnzlqa111110 opened the pull request:
first pass at adding urls to exclude that might otherwise return less reliable or useful pages in searches (how is the format? Wanted to check before adding another 100 haha)

pnzlqa111110 committed:
Update annotations.xml

first pass at adding urls to exclude that might otherwise return less reliable or useful pages in searches (how is the format? Wanted to check before adding another 100 haha)

nicolas-raoul commented:
Would you mind validating the XML file using an XML validator such as https://www.xmlvalidation.com ?

Using that validator shows this issue:

<img width="676" height="80" alt="Screenshot From 2025-10-18 11-26-56" src="https://github.com/user-attachments/assets/cdfaaebe-0fbb-499a-acb6-651ff212ee7d" />

As you can see, closing tag `</Annotation>` is present twice, you need to remove line 591.

Thanks! :-)

pnzlqa111110 committed:
Update annotations.xml

removed line 591 fixing error

nicolas-raoul commented:
Great moves and exclusions!

Nitpicking about spacing:
Would you mind checking https://github.com/pnzlqa111110/Wikipedia-Reliable-Sources/blob/patch-5/annotations.xml#L197 and fixing the indentation and extra white lines?
Do not hesitate to push commits and check again several times until it seems OK.

Thanks!

pnzlqa111110 committed:
Update annotations.xml

removed indentations

nicolas-raoul commented:
Last rogue whitespace: https://github.com/pnzlqa111110/Wikipedia-Reliable-Sources/blob/patch-5/annotations.xml#L269

pnzlqa111110 committed:
Update annotations.xml

removed last indentation - when i validated it it didn’t give me an error but good to know that doing without works best

nicolas-raoul commented:
Thanks! :-)
